notebooks/data-sources/TestGrid/metrics/number_of_flakes.ipynb

```diff
@@ -18,8 +18,13 @@
   },
```

Comment:
<p>that print statement was just to keep track of the phase of the loop. since you removed the loop from this cell, take out the print statement too. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/119/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
updated :+1: